packages/matchbox/src/components/Toggle/Toggle.js

```diff
@@ -1,68 +1,75 @@
-import React, { Component } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
-import classnames from 'classnames';
+import { visuallyHidden } from '../../styles/helpers';
+import styled from 'styled-components';
+import { deprecate } from '../../helpers/propTypes';
+import { ScreenReaderOnly } from '../ScreenReaderOnly';
+import { margin } from 'styled-system';
+import { createPropTypes } from '@styled-system/prop-types';
+import { toggle, input, StyledIndicator, StyledOutline } from './styles';
+import { pick } from '@styled-system/props';
+import { omit } from '../../helpers/systemProps';
 
-import styles from './Toggle.module.scss';
+const StyledToggle = styled('label')`
+  ${toggle}
+  ${margin}
+`;
 
-class Toggle extends Component {
-  static displayName = 'Toggle';
+const StyledInput = styled('input')`
+  ${visuallyHidden}
+  ${input}
+`;
 
-  static propTypes = {
-    id: PropTypes.string.isRequired,
-    checked: PropTypes.oneOfType([
-      PropTypes.bool,
-      PropTypes.string
-    ]),
-    compact: PropTypes.bool,
-    value: PropTypes.oneOfType([
-      PropTypes.bool,
-      PropTypes.string
-    ]),
-    disabled: PropTypes.bool,
-    onChange: PropTypes.func,
-    onFocus: PropTypes.func,
-    onBlur: PropTypes.func
-  };
+function Toggle(props) {
+  const {
+    id,
+    value,
+    checked,
+    disabled,
+    onChange,
+    onFocus,
+    onBlur,
+    label,
```

Comment:
This is a new prop for the old version of the component - I guess we shouldn't pass it on in that case? Should I add that to the acceptance criteria for the component wrapper?

Reply:
Also, for the Storybook stories, could you pass in some content here as an example? Might be good to also have stories for the `required` version